docs/spec/sanitations.md

```diff
@@ -83,13 +83,27 @@ This document records the sanitation done on top of the official OpenAPI specifi
 
    - **Reason**: This change addresses the payload binding failures occurring due to the inclusion of `expired_at` and `metadata` as required fields.
 
+6. **ListMessageObject schema: `first_id` and `last_id` attributes were made nullable (`nullable: true`)**:
+
+   - **Changed Schema**: `ListMessageObject`
+
+   - **Original**:
+      - `first_id`: `string`
+      - `last_id`: `string`
+
+   - **Updated**:
+      - `first_id`: `string?`
+      - `last_id`: `string?`
+
+   - **Reason**: This change is necessary to prevent binding errors that occur when there are no messages in the thread, as `first_id` and `last_id` need to be nullable in such cases.
+
 ---
 
 ## OpenAPI cli command
 
 The following command was used to generate the Ballerina client from the OpenAPI specification. The command should be executed from the repository root directory.
 
 ```bash
-bal openapi -i docs/spec/openapi.yaml --mode client --tags Assistants --license docs/license.txt -o ballerina
```

Comment:
Should this be `Assistants` or `Images` ?

Reply:
It should be Assistants. It is fixed now

Reply:
Fixed with [8cf206d2f805874d5aae9fea8dfddec5405c8dcb](https://github.com/ballerina-platform/module-ballerinax-openai.assistants/pull/8/commits/8cf206d2f805874d5aae9fea8dfddec5405c8dcb)